src/main/java/se/datasektionen/calypso/feeds/IcalFeed.java

```diff
@@ -55,16 +53,14 @@ public String renderIcsFeedEnglish() {
 		ical.setColor(CALENDAR_COLOR);
 		ical.setProductId(CALENDAR_EN_PRODUCT_ID);
 
-		var isReception = receptionRepository.get();
-
 		apiRepository
 				.allEvents()
 				.stream()
 				.filter(e ->
 					e.getEventStartTime() != null && e.getEventEndTime() != null &&
 					LocalDateTime.now().minusMonths(2)
 					.compareTo(e.getEventStartTime()) < 0 &&
-					!(isReception.getState() && e.isSensitive())
+					!(darkmode.getCurrent() && e.isSensitive())
```

Comment:
Here `.getCurrent()` is ran for all events that's at most 2 months old. (same for swedish ical)
But I guess this is simple to fix just by fetching the darkmode state once before the stream.

I don't know how often calendar/apps generally update their external calendars, but I guess it is possible this function in worst case is executed every time someone opens their calendar app? (feels unlikely though, it is probably cached in some nice way)